### PR TITLE
Fix priest quest Desperate Prayer

### PR DESCRIPTION
**Pull request by gouletr**
The priest quest Desperate Prayer was messed up a little, and one of them was missing. Somehow this was unnoticed for quite some time. The commit https://github.com/Questie/Questie/commit/f6580c8ec744d1dc8debd79b3364c0182021ed72 didn't even properly fix what was reported in https://github.com/Questie/Questie/issues/1477. This should fix them all.

- Add missing quest [Desperate Prayer (5640)](https://classic.wowhead.com/quest=5640/desperate-prayer)
- Fix quest [Desperate Prayer (5634)](https://classic.wowhead.com/quest=5634/desperate-prayer)
  - started by [High Priestess Laurena (376)](https://classic.wowhead.com/npc=376/high-priestess-laurena)
  - no objective text
  - fix exclusive to quests: 5635, 5636, 5637, 5638, 5639, 5640
- Fix quest [Desperate Prayer (5635)](https://classic.wowhead.com/quest=5635/desperate-prayer)
  - started by [Priestess Josetta (377)](https://classic.wowhead.com/npc=377/priestess-josetta)
  - fix exclusive to quests: 5634, 5636, 5637, 5638, 5639, 5640
- Fix quest [Desperate Prayer (5636)](https://classic.wowhead.com/quest=5636/desperate-prayer)
  - fix exclusive to quests: 5634, 5635, 5637, 5638, 5639, 5640
- Fix quest [Desperate Prayer (5637)](https://classic.wowhead.com/quest=5637/desperate-prayer)
  - started by [Maxan Anvol (1226)](https://classic.wowhead.com/npc=1226/maxan-anvol)
  - fix exclusive to quests: 5634, 5635, 5636, 5638, 5639, 5640
- Fix quest [Desperate Prayer (5638)](https://classic.wowhead.com/quest=5638/desperate-prayer)
  - fix exclusive to quests: 5634, 5635, 5636, 5637, 5639, 5640
- Fix quest [Desperate Prayer (5639)](https://classic.wowhead.com/quest=5639/desperate-prayer)
  - fix exclusive to quests: 5634, 5635, 5636, 5637, 5638, 5640

### CODE REVIEW

**Comment by Muehe** [comment]
Hey, thanks for the effort, we will surely merge it sooner or later. Just one small problem, the DB files shouldn't be edited at all. They are generated by a script, so when we update them any changes there will be overwritten.

If a quest is missing it has to be added as a correction, only difference being that you have to create a table first, like is already done for some other quests at the beginning of the `QuestieQuestFixes:Load()`function at the top of the file. You can fix that yourself or we will when we get around to merging.

Not sure why the build script failed, will look into it.

**Comment by gouletr** [comment]
> Just one small problem, the DB files shouldn't be edited at all.
Yup I knew that was going to be the case, thanks for the info I will fix it now.

File: Database/Corrections/Classic/classicQuestFixes.lua [hunk 1]
```diff
@@ -1621,7 +1623,38 @@ function QuestieQuestFixes:Load()
...
+        [5639] = {
+            [questKeys.exclusiveTo] = {5634,5635,5636,5637,5638,5640},
+        },
+        [5640] = {
+            [questKeys.name] = "Desperate Prayer",
             [questKeys.startedBy] = {{11401},nil,nil},
+            [questKeys.finishedBy] = {{376},nil},
+            [questKeys.requiredLevel] = 10,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = bit.bor(raceIDs.HUMAN, raceIDs.DWARF),
```
Comment (BreakBB) [inline Database/Corrections/Classic/classicQuestFixes.lua:1653]:
```suggestion
            [questKeys.requiredRaces] = raceIDs.HUMAN + raceIDs.DWARF,
```

This should be the correct format. The `or` connection is done in the code.

**Review by BreakBB — CHANGES_REQUESTED** [review]
Hey @gouletr Looks like you took a deep look into that quest and what is missing in our database. A big thank you, I will be happy to merge this after the small change 👍🏻

File: Database/Corrections/Classic/classicQuestFixes.lua (hunk 1 above)
Comment (gouletr) [reply Database/Corrections/Classic/classicQuestFixes.lua]: Thanks! I was wondering how to handle this.

**Review by BreakBB — APPROVED** [review]

**Pull request merged**